client/src/styles/Survey.styles.ts

```diff
@@ -63,7 +63,6 @@ export const pollTitleStyle = {
 };
 
 export const surveySubmitButtonStyles = {
-  width: '20rem',
```

Comment:
Curious why this is being removed?

Reply:
When the button was that wide I had the spinner come up on the extreme left while the text was aligned in center. It was visually jarring and this is better. Mentioned this change in button size to UX on the channel too.

Reply:
This piece of code works: 

![2022-11-02 14_12_51-PostCallOneQuestionPoll tsx - communication-services-virtual-visits-js-dailasm-s](https://user-images.githubusercontent.com/43504546/199603634-e0a5441a-9440-4b7d-b524-3c895afd1df8.png)

![2022-11-02 14_12_11-Book and 9 more pages - Work - Microsoft​ Edge](https://user-images.githubusercontent.com/43504546/199603649-6ad46206-fe0f-450b-a71f-0a2c8f2e6366.png)

This way the button width matches the text box width and also retains consistency with the Builder preview.

Reply:
Thanks! Have made this change